src/App.jsx

```diff
@@ -1,104 +1,32 @@
 import './App.scss';
-
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+
+function getUsers(userId) {
+  return usersFromServer.find(user => user.id === userId);
+}
+
+function getComments(postId) {
+  return commentsFromServer
+    .filter(comment => comment.postId === postId)
+    .map(comment => ({
+      ...comment,
+      user: getUsers(comment.userId),
+    }));
```

Comment:
- `getCommentsByPostId` describes better
- Also, do you actually need all user information in comment?